pkg/apis/kops/containerdconfig.go

```diff
@@ -67,11 +70,13 @@ type NRIConfig struct {
 
 type NvidiaGPUConfig struct {
 	// Package is the name of the nvidia driver package that will be installed.
-	// Default is "nvidia-headless-510-server".
+	// Default is "nvidia-driver-535-server".
 	DriverPackage string `json:"package,omitempty"`
 	// Enabled determines if kOps will install the Nvidia GPU runtime and drivers.
-	// They will only be installed on intances that has an Nvidia GPU.
+	// They will only be installed on instances that has an Nvidia GPU.
 	Enabled *bool `json:"enabled,omitempty"`
+	// Image defines the container image used to deploy the Nvidia Kubernetes Device Plugin.
+	K8sDevicePluginImage string `json:"image,omitempty"`
```

Comment:
Please rename this field, I think there's no need for the `K8s` prefix.
```suggestion
	DevicePluginImage string `json:"image,omitempty"`
```